expo/gbe/views/bid_act_view.py

```diff
@@ -144,7 +143,8 @@ def BidActView(request):
 
     else:
         form = ActEditForm(initial={'owner': profile,
-                                    'performer': personae[0]},
+                                    'performer': personae[0],
+                                    'b_conference': conference, },
```

Comment:
This refactor was necessary to let the 'validate_unique' check work on an act save - the check requires that the values are unique per conference.  With no default conference set for the Biddable, it means we have to set the conference manually during create.